vars/createTestJobPipeline.groovy

```diff
@@ -0,0 +1,90 @@
+#!groovy
+
+def call() {
+
+    def builder = getJenkinsLabels("aws", "eu-west-1")
```

Comment:
Shouldn't it be taken from params?

Reply:
i didn't put there, because this job will run only once for version and i think it is critical to set exact cloud and region in job.